javalib/src/main/scala/java/io/FilterReader.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package java.io
+
+abstract class FilterReader protected (in: Reader) extends Reader {
```

Comment:
```suggestion
abstract class FilterReader protected (protected val in: Reader) extends Reader {
```

Reply:
Does it need both protected?

Reply:
`protected (` means that the constructor is protected. `protected val` means that the member field is protected.

Reply:
The first is the ctor visibility then and the second exposes `in` as a protected `val` so it can be accessed in subclasses?

Reply:
Looked at the Javadocs and then it made sense - thanks.